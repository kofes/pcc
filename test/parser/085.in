type
	rc = record
		x,y:real;

