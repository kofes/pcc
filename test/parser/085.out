Unexpected end of file in pos (5, 1); expected 'END';
